activestorage/lib/active_storage/transformers/image_processing_transformer.rb

```diff
@@ -31,7 +31,7 @@ def operations
             if name.to_s == "combine_options"
               raise ArgumentError, <<~ERROR.squish
                 Active Storage's ImageProcessing transformer doesn't support :combine_options,
-                as it always generates a single ImageMagick command.
+                as it always generates a single command.
```

Comment:
I double checked the `combine_options` operation.
It does seem to be used by MiniMagick only:
https://github.com/minimagick/minimagick/blob/6a232e174fef0b67e35144ab87785cce1e190969/lib/mini_magick/image.rb#L473
I still think it makes sense to remove `ImageMagick` here for those using libvips.